raphtory/src/db/view_api/internal.rs

```diff
@@ -227,30 +227,6 @@ pub trait GraphViewInternalOps {
         layer: usize,
     ) -> Option<EdgeRef>;
 
-    /// Returns all the edge references in the graph.
-    ///
-    /// # Returns
-    ///
-    /// * `Box<dyn Iterator<Item = EdgeRef> + Send>` - An iterator over all the edge references.
-    fn edge_refs(&self, layer: Option<usize>) -> Box<dyn Iterator<Item = EdgeRef> + Send>;
-
-    /// Returns all the edge references in the graph created between the start (t_start) and
-    /// end (t_end) timestamps (inclusive).
-    ///
-    /// # Arguments
-    ///
-    /// * `t_start` - The start time of the window (inclusive).
-    /// * `t_end` - The end time of the window (exclusive).
-    /// # Returns
-    ///
-    /// * `Box<dyn Iterator<Item = EdgeRef> + Send>` - An iterator over all the edge references.
-    fn edge_refs_window(
-        &self,
-        t_start: i64,
-        t_end: i64,
-        layer: Option<usize>,
-    ) -> Box<dyn Iterator<Item = EdgeRef> + Send>;
-
```

Comment:
These methods should have been used by GraphViewOps `edges` for future optimisation (I am currently working on that...). I see that for some reason that is not the case. Can we make `GraphViewOps` use them instead of deleting them?

Reply:
I can leave them in, I don't think I'll use them, this is mean to be a small PR to move to GraphViewOps for the eval part